test/torchaudio_unittest/transducer_test.py

```diff
@@ -241,6 +242,14 @@ def compute_with_pytorch_transducer(data):
     return costs, gradients
 
 
+def skipIfNoRNNT(test_item):
+    try:
+        import torch.ops.torchaudio.rnnt_loss as _
+        return test_item
+    except ImportError:
```

Comment:
With the above change, this should be `RuntimeError`

```
>>> _ = torch.ops.torchaudio.rnn_loss
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/moto/conda/envs/PY3.8-cuda101/lib/python3.8/site-packages/torch/_ops.py", line 61, in __getattr__
    op = torch._C._jit_get_operation(qualified_op_name)
RuntimeError: No such operator torchaudio::rnn_loss
```

Reply:
l keep import because in my machine, if I only do `_ = torch.ops.torchaudio.rnn_loss` without import, it won't skip test. I think it's because when running on script, the code is not actually executed vs in interactive mode, so it wont trigger exception. but to pass the style check on github, I tried to do `_ = rnnt_loss`. let me know if you think of better alternative

Reply:
>  if I only do _ = torch.ops.torchaudio.rnn_loss without import, it won't skip test.

That might mean that you have a nightly-built torchaudio installed.
Can you verify that you do not have a torchaudio installed somewhere else?

Reply:
yes I do have the nightly-build installed, but I thought that's the instruction in the issue #1251. So should I remove the import part and only do `_ = torch.ops.torchaudio.rnn_loss`? Thank you!

Reply:
The direction is to install nightly-build of PyTorch and not nightly-build of torchaudio.
Anyways, please just do the attribute access only.

Reply:
got it, sorry about that, I misunderstand that point. will do so immediately